db/data/seedDatabase.js

```diff
@@ -12,22 +12,22 @@ MongoClient.connect(url, (err, client) => {
 
     let id = 0;
     let fakeRestaurantsBatch;
-    const batchSize = 10000;
+    const batchSizeSetting = 10000;
 
     const generateFakeRestaurantsBatch = (batchSize) => {
-      for (let i = 0; i < batchSize; i++) {
+      for (let i = 0; i < batchSize; i += 1) {
```

Comment:
does this need to refer to `batchSizeSetting` now that you changed the variable name above?

Reply:
I think this brings up an interesting point related to linting. When I call generateFakeRestaurantsBatch I pass in batchSizeSetting as an argument and it becomes batchSize in the function scope. I changed it to 'batchSizeSetting' because the airbnb linter complained that I used the same variable name for my parameter as I did in the closure scope. This is apparently called a shadow variable. I actually found it arguably harder to read after I changed it now that there are two variable names that essentially refer to the same thing though... https://eslint.org/docs/2.0.0/rules/no-shadow.